db/migrations/2023020100-init.php

```diff
@@ -299,6 +299,7 @@ public function up(): int
                 `is_banned` tinyint(1) unsigned NOT NULL DEFAULT 0 COMMENT '是否封禁',
                 `banned_reason` varchar(255) NOT NULL DEFAULT '' COMMENT '封禁理由',
                 `is_shadow_banned` tinyint(1) unsigned NOT NULL DEFAULT 0 COMMENT '是否处于账户异常状态',
+                `telegram_id` bigint(20) unsigned NOT NULL DEFAULT 0 COMMENT 'Telegram ID',
```

Comment:
This should not be added.

Reply:
Why?When I first installed and ran `php xcat Migration new`, it prompted that there is a missing`telegram_id` during the merge."